### DISCUSSION TITLE
React Native or Mobile Port

### DISCUSSION THREAD

**Discussion by satvikmaker** [saadeghi/daisyui#507] [General]
Hi Saadeghi,

The project looks impressive. Are you planning to (at any point of time), start a React Native or Mobile version of the same?
It would be amazing. In case you have plans, I would be happy to contribute as well.

Once again, congratulations on building an amazing components library.

**Comment by saadeghi** [comment]
I tried to make it work on React Native but the result did not look good. Global CSS variables didn't work and CSS child selectors didn't have the correct style. I don't have much experience in React Native so I'm not sure if it's a limitation of React Native or it needs some special config.

I'm not sure if there are any Mobile frameworks out there that would apply CSS class names for elements and fully support inheritance and CSS variables (like browsers do)  
If there's one, daisyUI would work on it.

**Comment by saadeghi** [comment]
I'm moving this to discussions.

**Reply by jorgebaralt** [reply]
Any plans on making this work? 👀

**Reply by saadeghi** [reply]
@jorgebaralt As I said, I don't have much experience in React Native. If anyone's willing to contribute on this, that would be great.

**Reply by dhamdani666** [reply]
I'm also waiting for the arrival for mobile , 
I will try the implementation myself with daisy ui for react native

**Comment by whck6** [comment]
Look forward to happening.

**Comment by dohomi** [comment]
Could daisyui get connected with https://github.com/marklawlor/nativewind ? This could make the tailwind classes work on all devices (web, native etc)

**Reply by SushritPasupuleti** [reply]
Seconded! DaisyUI could get a huge edge over the competition with this!

**Reply by dohomi** [reply]
check out the answer of Nativewind regarding this issue: https://github.com/marklawlor/nativewind/discussions/223#discussioncomment-3547752

**Reply by SushritPasupuleti** [reply]
> check out the answer of Nativewind regarding this issue: [marklawlor/nativewind#223 (reply in thread)](https://github.com/marklawlor/nativewind/discussions/223#discussioncomment-3547752)

Hey thanks for pointing me to it, just got down this rabbithole!
